loopback4/Dockerfile

```diff
@@ -1,13 +1,28 @@
-ARG node_version=12
```

Comment:
I think we want the old file here. I don't think I pushed the correct Dockerfile for screener compatibility, so this being replaced by the default Dockerfile from the loopback app generator.

Reply:
👍 good catch.